logging/log_storage/installing-log-storage.adoc

```diff
@@ -26,6 +26,48 @@ include::modules/loki-create-object-storage-secret-console.adoc[leveloffset=+2]
 .Additional resources
 * xref:../../logging/log_storage/installing-log-storage.adoc#logging-loki-storage_installing-log-storage[Loki object storage]
 
+[id="installing-log-storage-loki-sts"]
+== Deploying a Loki log store on an STS enabled cluster
```

Comment:
I don't know what the "default" for customers will be, but I would assume that when you create an STS-enabled cluster this does not automatically mean that your object storage will be ODF/Minio in the cluster, but still the "default object storage" of your public cloud (AWS S3, Azure Blob Storage).

This section explains how to "deploy Loki in a non-STS way on an STS-enabled cluster" which is a relevant thing to explain but comes out as "the recommended way to install Loki on an STS-enabled cluster" if we write it like this, which I would think is not what we want.

Reply:
I see what you mean, thanks.